crypto/ocsp/test/aws/OCSP-TEST.md

```diff
@@ -28,6 +28,26 @@ The leaf cert/key. OCSP responses will be generated for this cert.
 
 DER formatted OCSP response for the Server Cert.
 
+## Generating a OCSP request for the leaf cert
+```
+openssl ocsp -issuer ca_cert.pem -cert server_cert.pem -reqout ocsp_request.der
```

Comment:
I tried running this in `aws-lc/crypto/ocsp/test` and it failed with:
> Could not open file or uri for loading issuer certificate from ca_cert.pem

I see the readme mention ca_cert.pem, but it's not checked in and I can't find instructions to generate it

Reply:
Ah the file `ca_cert.pem` is under `aws-lc/crypto/ocsp/test/aws`. I placed it under the same directory as this `README` :). It's from our first iteration of OCSP work last year.